warehouse/query-core/src/main/java/datawave/query/jexl/DatawaveInterpreter.java

```diff
@@ -439,6 +439,7 @@ private void addHitsForFunction(String field, ASTFunctionNode node, HitListArith
     }
     
     public Object visit(ASTAndNode node, Object data) {
+        
```

Comment:
I love the simplification of this method.  awesome.  I wonder if the same thing can be done to the interpretOr method.  Just a thought, but not required of course.